controller/api/models.py

```diff
@@ -519,7 +536,6 @@ class Push(UuidAuditedModel):
     class Meta:
         get_latest_by = 'created'
         ordering = ['-created']
-        unique_together = (('app', 'uuid'),)
```

Comment:
The reasoning behind this is because Django Rest Framework **demands** the two keys to be present during a serializer's `.create()`, which in our case we do not create the `uuid` key until just before we [save it in the database](https://github.com/deis/deis/blob/c28e4ace5f22db14985a996aa00269aae96e7ea8/controller/api/fields.py#L31-L39).

From http://www.django-rest-framework.org/api-guide/serializers/:

> The ModelSerializer class is the same as a regular Serializer class, except that:

>- It will automatically generate a set of fields for you, based on the model.
>- It will automatically generate validators for the serializer, such as unique_together validators.
>- It includes simple default implementations of .create() and .update().

We could work around this by somehow disabling this specific UniqueTogether validation in the Serializers. Would that be reasonable?